R/features.R

```diff
@@ -62,6 +62,11 @@ feature_combinations <- function(m, exact = TRUE, n_combinations = 200, weight_z
     dt <- feature_exact(m, weight_zero_m)
   } else {
     dt <- feature_not_exact(m, n_combinations, weight_zero_m)
+    stopifnot(
+      data.table::is.data.table(dt),
+      !is.null(dt[["p"]])
+    )
+    p <- NULL # due to NSE notes in R CMD check
     dt[, p := NULL]
```

Comment:
I don't remember whether you answered this elsewhere, but why is this not handled within `feature_not_exact`?

Reply:
We have some decent unit tests on the probabilities, and I'd like to keep those for now (even though they are no longer use as the actual weights).